Mathlib/Topology/Order/IsLUB.lean

```diff
@@ -193,6 +214,23 @@ theorem exists_seq_tendsto_sSup {α : Type*} [ConditionallyCompleteLinearOrder 
   rcases (isLUB_csSup hS hS').exists_seq_monotone_tendsto hS with ⟨u, hu⟩
   exact ⟨u, hu.1, hu.2.2⟩
 
+theorem Dense.exists_seq_strictMono_tendsto' [DenselyOrdered α] [FirstCountableTopology α]
```

Comment:
I know that we already use primed lemmas in this file, but let's not increase the technical debt and use a more descriptive modifier, e.g.,
```suggestion
theorem Dense.exists_seq_strictMono_tendsto_of_lt [DenselyOrdered α] [FirstCountableTopology α]
```